code/cryptography/src/baconian_cipher/baconian.js

```diff
@@ -0,0 +1,21 @@
+
+function baconianEncrypt(plainText) {
+    baconianCodes = {
+        'a': 'AAAAA', 'b': 'AAAAB', 'c': 'AAABA', 'd':
+            'AAABB', 'e': 'AABAA', 'f': 'AABAB', 'g': 'AABBA', 'h': 'AABBB',
+        'i': 'ABAAA', 'j': 'BBBAA', 'k': 'ABAAB', 'l': 'ABABA', 'm':
+            'ABABB', 'n': 'ABBAA', 'o': 'ABBAB', 'p': 'ABBBA', 'q': 'ABBBB',
+        'r': 'BAAAA', 's': 'BAAAB', 't': 'BAABA', 'u': 'BAABB', 'v':
+            'BBBAB', 'w': 'BABAA', 'x': 'BABAB', 'y': 'BABBA', 'z': 'BABBB'
+    };
+
+    let cipherText = "";
+    [...plainText].forEach(char => {
```

Comment:
I would just omit the braces; `forEach` doesn't use the return value anyways.

Reply:
will u plz make it clear what do you want ?
because code is working fine at my system

Reply:
I have never done this just trying to learn how.after my last three phones got hacked and want to learn how I can stop it.and add more securury